Automattic/sensei
# Fix alignment of url buttons for Blockbase/Astra themes

Fixes https://github.com/Automattic/sensei/issues/5808
Depends on https://github.com/Automattic/sensei-pro/pull/2078

### Changes proposed in this Pull Request

* Replaced anchor tags with form and button so that we don't have to fight with styles in different themes.

### Testing instructions

- Activate the Astra theme and Sensei Pro should have https://github.com/Automattic/sensei-pro/pull/2078 checked out.
- Ensure there's a paid course that you aren't enrolled in, a course that you've completed, one you have made some progress in but not completed. They all should have at least some excerpts or description or something in the course body
- Ensure there's a course that you are enrolled in (free or paid).
- View the Course List block on the frontend and check all of those courses.
- Make sure the action buttons for all of them are aligned and look similar
- Make sure all of those buttons work as expected
- Check with the other popular themes for Sensei and make sure it hasn't broken in any of those

### Screenshot / Video
Blockbase -
<img width="1063" alt="image" src="https://user-images.githubusercontent.com/6820724/193011581-03ff8bad-6a1e-4a5e-942b-08ae6f442453.png">
Astra
<img width="1360" alt="image" src="https://user-images.githubusercontent.com/6820724/193011833-4b27d391-5ba4-481f-8b99-291d2813b3ce.png">
